androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/MultiChoiceForm.kt

```diff
@@ -0,0 +1,102 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.surveydetail
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import co.nimblehq.avishek.phong.kmmic.android.R
+import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.SurveyAnswerUiModel
+
+@Composable
+fun MultiChoiceForm(
+    surveyAnswerUiModels: List<SurveyAnswerUiModel>,
+    modifier: Modifier = Modifier,
+) {
+    var checkedAnswers by remember { mutableStateOf(emptySet<SurveyAnswerUiModel>()) }
+    LazyColumn(modifier = modifier.padding(horizontal = 80.dp)) {
+        items(surveyAnswerUiModels.size) { index ->
```

Comment:
It is a good practice to include `key` if you use `LazyColumn` to reduce the `recomposition` count.
```suggestion
        items(
            count = surveyAnswerUiModels.size,
            key = { index -> surveyAnswerUiModels[index].id },
        ) { index ->
```

Reply:
Updated here https://github.com/nimblehq/avishek-phong-kmm-ic/pull/125/commits/56152ad459b92c5a8576eb202c4f690396a43267